### DISCUSSION TITLE
Implementing AWSBatchTask class

### DISCUSSION THREAD

**Discussion by shinichi-takayanagi** [PrefectHQ/prefect#3216] [Ideas]
I think it would be very useful to have an AWSBatchTask like [Airflow](https://airflow.apache.org/docs/stable/_api/airflow/contrib/operators/awsbatch_operator/index.html) and [Luigi](https://luigi.readthedocs.io/en/stable/api/luigi.contrib.batch.html).

I've looked for the related topics on Slack and GitHub Issues, but it doesn't seem to have been considered yet.

Would it be a good contribution for Prefect if I implement it?

**Comment by joshmeek** [comment]
@shinichi-takayanagi yes this would absolutely be a great contribution! Feel free to reach out on here, community slack, or open a draft PR for some assistance along the way!
